### DISCUSSION TITLE
No thumbnails of HEIC files, how to repair it?

### DISCUSSION THREAD

**Discussion by jzaor** [4v3ngR/immich-native-macos#27] [Q&A] [answered]
I have uploaded a thousands photos (heic file format) from ios 18 device. This files has no thumbnails. 
I have tried yet:
1) Regenerate all thumbnails - none efect
2) brew reinstall heiflib (version 1.19.5), regenerate again - none efect.

I don’t know what to do to create heic’s thumbnails. Can you please help me?

**Comment by 4v3ngR** [comment]
Were you able to generate thumbnails with a preview version of immich?

**Reply by jzaor** [reply]
I think I don’t have a success with any previous versions. Today i am trying regenerating with the last version you published. (1.122.1) After regenerating there are no heic thumbnails again. I don’t know what to do. I see heic pictures in the immich app only using device where originally created by camera. On another device and web are only frames with information about an error during picture loading. That’s pity.

**Reply by jzaor** [reply]
1.120.1 - none heic thumbnails:
I tried to reinstall the imagemagick, vips and libheif by the brew.  I tried regenerate thumbnails with heic extension in file name. Nothing helps - none of heic thumbnails.

**Reply by jzaor** [reply]
1.119.1 - the same issue - no heic thumbnails.
No difference if the pictures are uploaded or from an external library

**Comment by 4v3ngR** [comment, marked as answer]
I've done some testing and I was able to upload some same heic files. Can you test 1.122.3 build B?

**Reply by jzaor** [reply]
It seems that it’s working! Thank you
